recipes/robotframework-interpreter/meta.yml

```diff
@@ -0,0 +1,41 @@
+{% set name = "robotframework-interpreter" %}
```

Comment:
please delete this duplicate file. The only difference seems to be the file extension.